src/components/MainContent/TaskModel.tsx

```diff
@@ -24,52 +25,63 @@ const renderDualTextOnlyStems = (stems: Stem[]) =>
   renderStemsBy('Appropriate Stems for Dual-Text Stimuli', stems);
 
 const renderFormatExample = (example: string, sectionName: string) => {
+  let elements: JSX.Element | undefined;
+
   // If example data has 'NA' or 'Examples', ignore them temporarily.
   // If API changes how to present none for example, this block should change.
   if (example === 'NA') return;
```

Comment:
Just check if `example` is NOT equal to `'NA'` and `'Examples`, then on the inside of that if loop, simple return the markup. any other code path in this scop will be a noop.
```
if(example === 'NA' || example === 'Examples'){
    return (<Section name={sectionName}>
      <SubHeader text={'Format Example'} />
      <Passage>{parseContent(example)}</Passage>
    </Section>)
}
```

Reply:
This has been updated.